vessel-runtime/src/test/java/com/textnow/android/vessel/VesselWrapper.kt

```diff
@@ -31,11 +55,11 @@ class VesselWrapper(private val vessel: Vessel, private val async: Boolean = fal
         }
     }
 
-    suspend fun preload(timeoutMS: Int?) {
+    suspend fun preload(timeoutMS: Int?): PreloadReport {
```

Comment:
This syntax can be cleaned up. a single `return if` would be better; or ideally `: PreloadReport = if`